### ISSUE TITLE
Class number out of index and incorrect annotations in Annotation/Test/Sunny/Night folder

### ISSUE THREAD

**Issue by srujanaaac** [Sourajit-Maity/juvdv2-vdvwc#3]
There are two issues with the annotations in the Annotation/Test/Sunny/Night folder:

1)The class number is given as 14 in several files, which is causing an index out of bounds error because the number of classes is only 14 (assuming the class numbers should be from 0 to 13).

2)The classes present in the annotations are different from the objects in the photos. For example let us take night(1) image , object 0 is labeled as a car and class 14, which is supposed to be class 13, is labeled as a boat. However, the image is of a city road at night with a rickshaw.

Attached is a file showing examples of the annotation files with the class number issue and discrepancies between annotated classes and objects in the photos.

![Git hub issue](https://github.com/Sourajit-Maity/juvdv2-vdvwc/assets/115392240/f0647ef2-6b10-44f8-b538-241e7c0e94cd)

**Comment by kirpurva** [comment]
There are actually 15 classes, not 14, I also faced this issue and had to debug this. If you look carefully, you'll find the 15th class labelled as class-id:14(since the id ranges from 0-14) in some annotation files . The class names are: ['car', 'bike', 'auto', 'rickshaw', 'cycle', 'bus', 'minitruck', 'truck', 'van', 'taxi', 'motorvan', 'toto', 'train', 'boat', 'cycle van']. And yes the dataset is very poorly curated, but I suppose the challenge lies there only. I thus  recommend you to configure the driver code to accept 15 classes and not 14 in order to fix this.

**Comment by srujanaaac** [comment]
Yes, I have looked through it again and found this out. I agree there few images where van is labelled as car, along with it I presumed cycle car as Rikshaw, when I went through obj.data I found it to be separate class named cycle car.
